<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QComicsViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFirst_Page"/>
    <addaction name="actionLast_Page"/>
    <addaction name="actionPrev_Page"/>
    <addaction name="actionNext_Page"/>
    <addaction name="actionGoto_Page"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Double_Pages"/>
    <addaction name="actionSingle_First_Page"/>
    <addaction name="separator"/>
    <addaction name="actionFit_Width"/>
    <addaction name="actionFit_Height"/>
    <addaction name="actionFit_Page"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionFirst_Page"/>
   <addaction name="actionPrev_Page"/>
   <addaction name="actionNext_Page"/>
   <addaction name="actionLast_Page"/>
  </widget>
  <action name="actionNext_Page">
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionPrev_Page">
   <property name="text">
    <string>Prev Page</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="text">
    <string>Last Page</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="text">
    <string>First Page</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionShow_Double_Pages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Double Pages</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGoto_Page">
   <property name="text">
    <string>Goto Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="actionFit_Height">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Height</string>
   </property>
  </action>
  <action name="actionFit_Page">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Page</string>
   </property>
  </action>
  <action name="actionSingle_First_Page">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Single First Page</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
